openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -82,6 +85,20 @@ class SMIRNOFFSpecError(MessageException):
     pass
 
 
+class FractionalBondOrderInterpolationMethodUnsupportedError(MessageException):
+    """
+    Exception for when an unsupported fractional bond order interpolation assignment method is called.
+    """
+
+    pass
```

Comment:
Not really consequential, but these `pass` lines unnecessary/for readability only. As long as a class has a docstring, it works

```python3
>>> class PointlessClass:
...
  File "<stdin>", line 2

    ^
IndentationError: expected an indented block
>>> class PointlessClass:
...     """Docstrings for pointless classes? Sure, why not"""
...
>>> PointlessClass()
<__main__.PointlessClass object at 0x7f9fb4366eb8>

Reply:
Oh, silly me. I'll leave them for consistency with the widespread (mis)use currently in the codebase, but I'd welcome a separate PR to pull all instances of this out at once.